Content.Server/Disposal/Unit/EntitySystems/DisposalUnitSystem.cs

```diff
@@ -285,6 +285,7 @@ private void HandleThrowCollide(EntityUid uid, DisposalUnitComponent component,
                 _robustRandom.NextDouble() > 0.75 ||
                 !component.Container.Insert(args.Thrown))
             {
+                args.Thrown.PopupMessageEveryone(Loc.GetString("disposal-unit-thrown-missed"));
```

Comment:
```suggestion
                if(args.User.HasValue)
                    _popupSystem.PopupEntity(Loc.GetString("disposal-unit-thrown-missed"), args.Thrown, Filter.Entities(args.User.Value));
```

don't think the missed message needs to be sent to anyone but the user. the if is to ensure there was a user that threw it since it is nullable

Reply:
I think I prefer it going to everyone in range. It's much clearer to all observers what's going on (i.e. it dosen't look like a bug like described in #10377). Since this isn't a "private" event (i.e. it's not about your own body or held item) making it public makes more sense to me.

Reply:
use the `_popupSystem` methods still rather than the entityuid override

Reply:
```suggestion
                _popupSystem.PopupEntity(Loc.GetString("disposal-unit-thrown-missed"), args.Thrown, Filter.Pvs(args.User.Value));
```